NewRelicBrowser.php

```diff
@@ -1,392 +1,456 @@
 <?php
-
 /**
+ * New Relic Browser Plugin developed by rtCamp
+ *
+ * @package     NewRelicBrowser
+ * @author      rtCamp
+ * @license     GPL-2.0+
+ *
  * Plugin Name: New Relic Browser
- * Plugin URI: http://www.rtcamp.com
+ * Plugin URI:  http://www.rtcamp.com
```

Comment:
https://rtcamp.com

Reply:
i solved this issue. please review changes.